components/input.css

```diff
@@ -0,0 +1,26 @@
+.display-input{
+    display: flex;
+    flex-direction: column;
+    width: 20rem;
+}
+
+.input{
+    margin-left: 1rem;
+    padding: 1rem;
+}
+
+.input-label{
+    margin: 1rem;
+}
+
+.input-label-success{
+    border: 2px solid var(--success);
+}
+
+.input-label-warning{
+    border: 2px solid var(--warning);
+}
+
+.input-label-error{
+    border: 2px solid var(--danger);
+}
```

Comment:
just if you are not aware, you can even use CSS variable for entire border property like `border: var(--input-border)` so that you can even change thickness of border like color.

Reply:
Thanks for letting me know about it.